astarte/device/introspection.py

```diff
@@ -25,9 +26,9 @@ class Introspection:
     name but different major/minor are not supported.
     """
     def __init__(self):
-        self.__interfaces_list = {}
+        self.__interfaces_list: dict = {}
```

Comment:
```suggestion
        self.__interfaces_list = {}
```
I think it's kind of implied by the initialization.